src/build.c

```diff
@@ -934,20 +924,17 @@ static gchar *prepare_run_script(GeanyDocument *doc, gchar **vte_cmd_nonscript,
 	}
 #endif
 
-	/* RUN_SCRIPT_CMD should be ok in UTF8 without converting in locale because it
-	 * contains no umlauts */
```

Comment:
But working_dir is in UTF8 so there would be a problem. Anyway, we get the file name now in locale from the tmpfile creation function so this part is irrelevant now.